config/default.json

```diff
@@ -30,5 +30,11 @@
         }
       }
     }
+  },
+  "logger": {
+    "level": "debug",
```

Comment:
I feel like the default should probably be "important" so that 'debug' can have more verbose logs that don't show up by default and have to be opted-into

Reply:
Yeah I was also unclear about what "default" meant and when this json would actually be used.

I guess my idea was we should keep debug on in development?

Reply:
since I made this comment 5 hours ago I have since come to the conclusion that the exact values of `default.json` don't actually matter, just its structure does as that's used to autogenerate the `Config.d.ts` file which defines the type of the `config` object within the code.  So I think it probably doesn't matter what we put here so long as it's a string.  Would be nice to confirm that though... @oed?

Reply:
So I just did some manual testing and I was wrong - the values in default.json do indeed get used as default values if there is no `.env` file present, or for any fields that the `.env` file doesn't specify.

Reply:
We also have example files `config/env/{dev,prod,test}.json` where we can put different recommended defaults for different environments.  But what's set here will be the primary default if nothing else is configured.

Reply:
cool so will change to `important`